scripts/jenkins/ardana/ansible/roles/input_model_generator/templates/input_model/data/network_groups.yml

```diff
@@ -105,17 +117,26 @@
         #
 
         - provider: ip-cluster
+{%     if ('MANAGEMENT' and 'IPV4')  in network_group['component_endpoints'] %}
+          name: internal-lb-ipv4
+          tls-components:
+            - mysql
+            - rabbitmq
+          cert-file: ardana-internal-ipv4-cert
+{%     else %}
           name: internal-lb
           tls-components:
             - default
```

Comment:
It seems, default component is added into multiple network group MANAGEMENT & IPv4-MANAGEMENT. CP wont allow it.

Reply:
yeap. fixing this.

Reply:
Although in this particular part of the code it is not put in multiple places because it is an if/else. But it is further up above where the network groups are defined.